test/phx_tools_web/live/phx_tools_live_test.exs

```diff
@@ -0,0 +1,55 @@
+defmodule PhxToolsWeb.PhxToolsLiveTest do
+  use PhxToolsWeb.ConnCase, async: true
+
+  import Phoenix.ConnTest
+  import Phoenix.LiveViewTest
+
+  describe "landing page" do
+    test "user visits the page", %{conn: conn} do
```

Comment:
Add the assertation for the confirmation message and that the background color class is added to one of the platforms.

Reply:
Only add the assertation. No need for a new test case.